lib/elliptic/curve/base.js

```diff
@@ -243,6 +242,52 @@ BasePoint.prototype.validate = function validate() {
   return this.curve.validate(this);
 };
 
+BaseCurve.prototype.decodePoint = function decodePoint(bytes, enc) {
+  bytes = utils.toArray(bytes, enc);
+
+  if (this.type !== 'mont') {
```

Comment:
Now that we are on the `Curve` object, couldn't you then specialize this per curve to avoid this branch?
May not be worth it for the extra SLOC,  but,  still.